server/utils/auth.js

```diff
@@ -0,0 +1,37 @@
+const jwt = require('jsonwebtoken');
+
+const secret = 'scalamoushhhhhh';
+const expiration = '2h';
```

Comment:
We should add a .env file to store the secret in.

Reply:
I think you should keep the const secret variable and set it equal to process.env.JWT_SECRET.  `const secret = process.env.JWT_SECRET`;  Then on line 25 & 36 you would access it as you had it before as secret.